crates/sui-framework/packages/sui-framework/sources/random.move

```diff
@@ -129,6 +124,6 @@ module sui::random {
         new_bytes: vector<u8>,
         ctx: &TxContext,
     ) {
-        update_randomness_state(self, new_round, new_bytes, ctx);
+        self.update_randomness_state(new_round, new_bytes, ctx);
```

Comment:
Not sure what happened below but there seems to be code missing?

Reply:
Where?

Reply:
OK, this is really weird -- in my PR review (using a github client inside vim) it showed basically the whole rest of this file deleted...but that's obviously not the case now is it...